src/components/CloudBrute/CloudBrute.tsx

```diff
@@ -0,0 +1,173 @@
+import { Button, Stack, TextInput } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { useCallback, useState, useEffect } from "react";
+import { CommandHelper } from "../../utils/CommandHelper";
+import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { RenderComponent } from "../UserGuide/UserGuide";
+import { SaveOutputToTextFile } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
+import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
+import InstallationModal from "../InstallationModal/InstallationModal";
+
+/**
+ * Represents the form values for the CloudBrute component.
+ */
+interface FormValuesType {
+    domain: string;
+    keyword: string;
+    wordlist: string;
+}
+
+/**
+ * The CloudBrute component.
+ * @returns The CloudBrute component.
+ */
+const CloudBrute = () => {
+    // Component State Variables.
+    const [loading, setLoading] = useState(false); // State variable to indicate loading state.
+    const [output, setOutput] = useState(""); // State variable to store the output of the command execution.
+    const [pid, setPid] = useState(""); // State variable to store the process ID of the command execution.
+    const [isCommandAvailable, setIsCommandAvailable] = useState(false); // State variable to check if the command is available.
+    const [opened, setOpened] = useState(!isCommandAvailable); // State variable that indicates if the modal is opened.
+    const [loadingModal, setLoadingModal] = useState(true); // State variable to indicate loading state of the modal.
+
+    // Component Constants.
+    const title = "CloudBrute"; // Title of the component.
+    const description =
+        "CloudBrute is a tool for cloud enumeration and infrastructure discovery in various cloud providers."; // Description of the component.
+    const steps =
+        "Step 1: Enter the target domain.\n" +
+        "Step 2: Enter a keyword for URL generation.\n" +
+        "Step 3: Specify the path to the wordlist file.\n" +
+        "Step 4: Click 'Run CloudBrute' to start the scan.\n" +
+        "Step 5: View the Output block below to see the results of the scan.";
+    const sourceLink = ""; // Link to the source code.
```

Comment:
You forgot to add the sourcelink

```suggestion
    const sourceLink = "https://www.kali.org/tools/cloudbrute/"; // Link to the source code.
```